cpp/include/cudf/io/orc.hpp

```diff
@@ -445,24 +445,24 @@ class orc_writer_options {
   /**
    * @brief Whether writing column statistics is enabled/disabled.
    */
-  bool enable_statistics() const { return _enable_statistics; }
+  bool is_enabled_statistics() const { return _enable_statistics; }
 
   /**
    * @brief Returns maximum stripe size, in bytes.
    */
-  auto stripe_size_bytes() const { return _stripe_size_bytes; }
+  auto get_stripe_size_bytes() const { return _stripe_size_bytes; }
```

Comment:
I remember a while back, @codereport and @jrhemstad had a long discussion on whether `get` prefix should be used or not and the outcome is that no prefix is more readable IIRC.

Reply:
That's true in general. However, we made a conscious decision to use `get_` prefix for the `_options` classes. I'm unable to find the discussion that led to this (perhaps Jake had the discussion with Conor later on).
We can choose to remove the prefix, but it should be done for all options. Either way, I prefer to keep the API consistent, so I'm adding these options with the prefix.